src/components/RegistrationTable/EditCell.tsx

```diff
@@ -32,18 +57,53 @@ export const EditCell: React.FC<EditCellProps> = ({ row }) => {
                 </Button>
             </DialogTrigger>
 
-            <DialogContent className="max-w-md w-full bg-events-active-tab-bg">
+            <DialogContent className="max-w-[750px] w-full max-h-lg bg-events-active-tab-bg border-0">
                 <DialogHeader>
                     <DialogTitle className="text-white">{row.original.firstName} {row.original.lastName}</DialogTitle>
-                    <span className="italic">Form Responses</span>
+                    <span className="italic text-white">Form Responses</span>
                 </DialogHeader>
                 {/* divider */}
-
-                <div className="w-full h-[1px] bg-[#8DA1D1] my-3"/>
-
-                <Button onClick={handleEdit}>Save changes</Button>
-
+                <div className="max-w-full h-[1px] bg-[#8DA1D1] my-3" />
+                <div className="max-h-[300px] max-w-full overflow-y-auto">
+                    <UserInfo row={row} table={table} />
+                    <div className="max-w-full h-[1px] bg-[#8DA1D1] my-3">
+                    <UserResponses questions={questions} responses={responses} />
+                    </div>
+                </div>
+                <div className="w-full h-[1px] bg-[#8DA1D1] my-3" />
```

Comment:
If we're using the colour #8DA1D1 several times, could we add it to the styling sheet?